src/meta/src/hummock/manager/commit_epoch.rs

```diff
@@ -419,3 +443,46 @@ fn on_handle_add_new_table(
 
     Ok(())
 }
+
+/// Rewrite the commit sstables to sub-levels based on the compaction group config.
+/// The type of `compaction_group_manager_txn` is too complex to be used in the function signature. So we use `HashMap` instead.
+fn rewrite_commit_sstables_to_sub_level(
```

Comment:
`correct_commit_ssts` which generates`commit_sstables`, should ensure the relateive SST order is correct.
Rest LGTM.